### ISSUE TITLE
Link to t.p.logging in t.logger narrative docs

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8154]
|[<img alt="twm's avatar" src="https://avatars.githubusercontent.com/u/43662?s=50" width="50" height="50">](https://github.com/twm)| @twm reported|
|-|-|
|Trac ID|trac#8154|
|Type|defect|
|Created|2015-12-23 19:59:45Z|
|Branch|https://github.com/twisted/twisted/tree/logger-docs-8154|

Looks like a link was missed when the module was renamed.

Attachments:

* [8154-log-typo.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/379/379632b41163ac4c6f5ff6622a09e4f6f4ccd9c8/401acb3e46a10006c102046b54670af7494d5830.patch) (1012 bytes) - added by twm on 2015-12-23 20:02:51Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__8154 8154
type__defect defect
reporter__twm twm
priority__low low
milestone__None None
branch__branches_logger_docs_8154 branches/logger-docs-8154
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__Easy Easy
time__1450900785491834 1450900785491834
changetime__1451047671536464 1451047671536464
version__None None
owner__adiroiban adiroiban

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

(In [46546]) Branching to logger-docs-8154.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

Many thanks for your patch.

It looks good.

I have found another place where the old docs are referenced and I have updated it as part of this patch.

Leaving this for review so that another dev can also review my changes.

Patch applied, news file fragments created, branch sent to buildbot.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @adiroiban|
|-|-|

Looks good to me; please merge.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set status to `closed`|
|-|-|

(In [46570]) Merge logger-docs-8154: Update links from t.p.logging to t.logger in narrative docs

Author: twm, adiroiban
Reviewer: adiroiban, glyph
Fixes: [#8154](https://github.com/twisted/twisted/issues/8154)
